Containerfile

```diff
@@ -47,6 +47,16 @@ ENV BASH_ENV="${APP_ROOT}/venv/bin/activate" \
     PROMPT_COMMAND=". ${APP_ROOT}/venv/bin/activate" \
     PATH="${APP_ROOT}/venv/bin:${PATH}"
 
+COPY scripts/download_embeddings_model.py .
+RUN source ${APP_ROOT}/venv/bin/activate && python download_embeddings_model.py embeddings_model \
+    &&  rm download_embeddings_model.py
+
+COPY scripts/generate_embeddings.py .
+RUN source ${APP_ROOT}/venv/bin/activate \
+    && git clone https://github.com/ilan-pinto/lightspeed-rag-documents.git \
```

Comment:
we shouldn't need to do this, we should intend to treat the rag docs repo as our "source" and setup a CI build that clones that source and injects it into the build, the same way we build our other images from repos.  

And we should be doing this in a separate containerfile that just produces an image containing nothing but the embeddings.


ultimately this probably means:
1) we create a new github repo for the rag docs
2) we put this containerfile in that repo
3) we setup a new CI build that builds the rag content image from that repo
4) we update the OLS image build process to pull in the embeddings from the image produced by (3)